src/playground/sample_data_tooling/sample_data_tests/sample_data_ingestor_tests/test-files/ecommerce.json

```diff
@@ -0,0 +1,50 @@
+{"timestamp": 100000, "order_id": 1, "items_purchased_success": 6, "items_purchased_failure": 0, "total_revenue_usd": 79.73}
+{"timestamp": 133399, "order_id": 2, "items_purchased_success": 1, "items_purchased_failure": 0, "total_revenue_usd": 66.00}
```

Comment:
this json file should be as following, do you see the error for the format? "end of file expected on the line 2?"
[
    {},
    {},
    ....
    {}
]

Reply:
Changed `.json` to `.ndjson `